pkg/analyzer/analyzers/gitlab/gitlab.go

```diff
@@ -26,18 +29,80 @@ type Analyzer struct {
 func (Analyzer) Type() analyzerpb.AnalyzerType { return analyzerpb.AnalyzerType_GitLab }
 
 func (a Analyzer) Analyze(_ context.Context, credInfo map[string]string) (*analyzers.AnalyzerResult, error) {
-	_, err := AnalyzePermissions(a.Cfg, credInfo["key"])
+	key, ok := credInfo["key"]
+	if !ok {
+		return nil, errors.New("key not found in credentialInfo")
+	}
+
+	info, err := AnalyzePermissions(a.Cfg, key)
 	if err != nil {
 		return nil, err
 	}
-	return nil, fmt.Errorf("not implemented")
+	return secretInfoToAnalyzerResult(info), nil
+}
+
+func secretInfoToAnalyzerResult(info *SecretInfo) *analyzers.AnalyzerResult {
+	result := analyzers.AnalyzerResult{
+		AnalyzerType: analyzerpb.AnalyzerType_GitLab,
+		Metadata: map[string]any{
+			"version":    info.Metadata.Version,
+			"enterprise": info.Metadata.Enterprise,
+		},
+		Bindings: []analyzers.Binding{},
+	}
+
+	// Add token and it's permissions to bindings
+	tokenResource := analyzers.Resource{
+		Name:               info.AccessToken.Name,
+		FullyQualifiedName: fmt.Sprintf("github.com/token/%d", info.AccessToken.ID),
+		Type:               "access_token",
+		Metadata: map[string]any{
+			"last_used_at": info.AccessToken.LastUsedAt,
```

Comment:
Since we don't update resources, this value can easily become stale. I think it's best to remove it.

Reply:
removed.